src/traits.rs

```diff
@@ -60,11 +60,11 @@ where
     fn generate_safe_prime_with_rng<R: CryptoRng + ?Sized>(rng: &mut R, bit_length: u32) -> Self {
         generate_safe_prime_with_rng(rng, bit_length)
     }
-    fn is_prime_with_rng<R: CryptoRng + ?Sized>(&self, rng: &mut R) -> bool {
-        is_prime_with_rng(rng, self)
+    fn is_prime_with_rng<R: CryptoRng + ?Sized>(&self, _rng: &mut R) -> bool {
+        is_prime(self)
     }
-    fn is_safe_prime_with_rng<R: CryptoRng + ?Sized>(&self, rng: &mut R) -> bool {
-        is_safe_prime_with_rng(rng, self)
+    fn is_safe_prime_with_rng<R: CryptoRng + ?Sized>(&self, _rng: &mut R) -> bool {
+        is_safe_prime(self)
```

Comment:
So these are here for backwards compatibility? If yes, then what do you say about adding a deprecation notice?

```rust
#[deprecated(since = "0.8", note = "primality tests are now deterministic")]
```

Reply:
More like, to make things compile. I'm planning to remove this trait completely as a part of #62.